feature/networkinstance/networkinstance.go

```diff
@@ -49,9 +51,24 @@ func (ni *NetworkInstance) validate() error {
 	if ni.oc.GetType() == oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_UNSET {
 		return errors.New("NetworkInstance type is unset")
 	}
+	if ni.oc.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "static").GetOrCreateStatic("prefix").GetOrCreateNextHop("0").NextHop == oc.UnionString("") {
+		return errors.New("NetworkInstance nexthop is unset")
+	}
 	return ni.oc.Validate()
 }
 
+// WithStaticRoute sets the prefix value for static route.
+func (ni *NetworkInstance) WithStaticRoute(prefix string, nextHops []string) *NetworkInstance {
+	ni.oc.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "static").GetOrCreateStatic("prefix").Prefix = ygot.String(prefix)
+	static := ni.oc.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "static").GetOrCreateStatic(prefix)
```

Comment:
Hmm, this is a repeat of what you have before and you likely wanted to just do static.Prefix = ... in the earlier line. Here I see that you have the GetOrCreateStatic(prefix) correct.

Reply:
Done!